protobuf-provider/src/test/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchemaTest.java

```diff
@@ -621,6 +621,170 @@ public void testComplexCustomOptions2() {
     assertEquals(canonicalString, schema.canonicalString());
   }
 
+  @Test
+  public void testNestedCustomOptions() {
```

Comment:
Could this unit test be simplified to just test the Options kind so it's more obvious what the test case is doing?

Reply:
Actually this schema is already trimmed down significantly from the original.